src/modules/user/containers/pro-book-actions/pro-book-actions.tsx

```diff
@@ -43,15 +60,30 @@ export const ProBookActions: FC<ProBookActionsProps> = () => {
   return (
     <BottomFixedContent.Item orderIndex={1}>
       <div className={styles.root}>
-        <Button
+        {/* <Button
           className={styles.action}
           variant='secondary'
           text={intl.formatMessage({ id: 'button.contact' })}
+        /> */}
+        <UserContactPopup
+          professional={professional}
+          isOpen={isContactOpen.value}
+          onClose={isContactOpen.setFalse}
+          trigger={
+            <Button
+              variant='secondary'
+              text='Contact'
+              onClick={isContactOpen.setTrue}
+            />
```

Comment:
display this popup on top of the button. There is `align` props in Dialog component